packages/website/src/components/SidebarLayout.tsx

```diff
@@ -9,7 +9,7 @@ export function SidebarLayout({
 }: PropsWithChildren<Partial<ItemListProps & { data: { member: ReturnType<typeof findMember> } }>>) {
 	return (
 		<div className="flex flex-col lg:flex-row overflow-hidden max-w-full h-full max-h-full bg-white dark:bg-dark">
-			<div className="w-full lg:max-w-1/4 lg:min-w-1/4">
+			<div className="w-full lg:max-w-[320px] lg:min-w-[320px]">
```

Comment:
?

Reply:
The sidebar doesn't need to be stretch or compress in large breakpoints, only the content should be doing that.

Reply:
320 might still be a bit too tiny.